### ISSUE TITLE
Bug - Label group: vertical category with overflow produces wrong tab order

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#5122]
**Describe the problem**
The React implementation of the label group has the [Vertical category overflow removable](https://www.patternfly.org/v4/components/label-group#vertical-category-overflow-removable) example, which has an incorrect tab order based on its visual order. When tabbing, focus goes to the truncated category name, the "2 more" button, then back to the top for the remove button.

While Core doesn't have this example, the CSS would most likely need to be updated slightly so that the markup can have the "remove" button come before the list of labels in DOM order for the vertical variant, while still appearing horizontally to the right of the category name.

**How do you reproduce the problem?**
Provide steps to reproduce. A [codepen](http://codepen.io) or [codesandbox](https://codesandbox.io/) demonstrating the problem is appreciated.

**Expected behavior**
Tabbing would cause the focus order to be category name > remove button > overflow button.

**Comment by thatblindgeye** [comment]
Closing per conversation in #5172
